### PR TITLE
NSTask - capture output from commands

### PR DESCRIPTION
**Pull request by Bouke**

### CODE REVIEW

File: Foundation/NSTask.swift [hunk 1]
```diff
@@ -151,9 +151,27 @@ public class NSTask : NSObject {
     public var currentDirectoryPath: String = NSFileManager.defaultInstance.currentDirectoryPath
     
     // standard I/O channels; could be either an NSFileHandle or an NSPipe
-    public var standardInput: AnyObject?
-    public var standardOutput: AnyObject?
-    public var standardError: AnyObject?
+    public var standardInput: AnyObject? {
+        willSet {
+            guard newValue is NSPipe || newValue is NSFileHandle else {
+                abort()
```
Comment (parkera) [inline Foundation/NSTask.swift]: Let's change this (and following) to `preconditionFailure("message")` instead, so a caller knows what they did wrong without digging into the source code.

**Comment by parkera** [comment]
@swift-ci please test

**Comment by Bouke** [comment]
Ah yes, that's way better. I've updated the PR.

**Comment by parkera** [comment]
Looks good, thanks!

**Comment by parkera** [comment]
@swift-ci please test

**Pull request merged**